README.rst

```diff
@@ -307,28 +306,31 @@ Library
     ...    settings=settings
     ... ).filter_by_station_id(station_id=(1048, 4411))
     >>> request.df.head()  # station list
-        station_id                 from_date                   to_date  height  \
-    ...      01048 1934-01-01 00:00:00+00:00 ... 00:00:00+00:00   228.0
-    ...      04411 1979-12-01 00:00:00+00:00 ... 00:00:00+00:00   155.0
-    <BLANKLINE>
-         latitude  longitude                    name    state
-    ...   51.1278    13.7543       Dresden-Klotzsche  Sachsen
-    ...   49.9195     8.9671  Schaafheim-Schlierbach   Hessen
+    shape: (2, 8)
+    ┌────────────┬──────────────┬──────────────┬────────┬──────────┬───────────┬─────────────┬─────────┐
+    │ station_id ┆ from_date    ┆ to_date      ┆ height ┆ latitude ┆ longitude ┆ name        ┆ state   │
+    │ ---        ┆ ---          ┆ ---          ┆ ---    ┆ ---      ┆ ---       ┆ ---         ┆ ---     │
+    │ str        ┆ datetime[μs] ┆ datetime[μs] ┆ f64    ┆ f64      ┆ f64       ┆ str         ┆ str     │
+    ╞════════════╪══════════════╪══════════════╪════════╪══════════╪═══════════╪═════════════╪═════════╡
+    │ 01048      ┆ 1934-01-01   ┆ ...          ┆ 228.0  ┆ 51.1278  ┆ 13.7543   ┆ Dresden-Klo ┆ Sachsen │
+    │            ┆ 00:00:00     ┆ 00:00:00     ┆        ┆          ┆           ┆ tzsche      ┆         │
+    │ 04411      ┆ 1979-12-01   ┆ ...          ┆ 155.0  ┆ 49.9195  ┆ 8.9671    ┆ Schaafheim- ┆ Hessen  │
+    │            ┆ 00:00:00     ┆ 00:00:00     ┆        ┆          ┆           ┆ Schlierbach ┆         │
+    └────────────┴──────────────┴──────────────┴────────┴──────────┴───────────┴─────────────┴─────────┘
```

Comment:
Detecting differences within this table when values change may turn out to be difficult. Are there any other representations available?

Reply:
What about `format=NOTHING`?

Reply:
It looks more similar to what we have right now but the default version just looks better (isn't that important for the readme?)

Reply:
Ah, it's the README! Apologies, I missed that.